src/metabase/api/cache.clj

```diff
@@ -62,6 +62,19 @@
                                     "question"  :model/Card)
                                   :id [:in ids]))))
 
+(defn- check-cache-access [models id]
```

Comment:
nit: I think this would be neater if the signature of the function was `[model id]` not `[models id]` because it only checks access for the first element of `models`.

I'm imagining a call like `(check-cache-access ["database" "question"] 1)`. It would only execute `(api/write-check :model/Database 1)` and it would throw away `"question"`. We don't need to check multiple models here, so you might as well change the signature so the call was like this: `(check-cache-access ["database" "question"] 1)`.

Reply:
Is now better?